utils/azure_users.py

```diff
@@ -169,3 +169,37 @@ def get_test_user_ids(self):
         assert 200 == response.status_code
         assert 'value' in response.json()
         return [item['objectId'] for item in response.json()['value']]
+
+    def get_group_id_by_group_name(self, group_name):
+        endpoint_get_groups = "{endpoint}/groups?api-version=1.6&$select=objectId&$filter=displayName eq '{group_name}'".format(
```

Comment:
I think we could change the name of variable from `endpoint_get_groups` to `endpoint`. Also `response_get_groups` to `response`

Reply:
Sorry i forgot to change that variables, now i have changed it.